libs/networking/http_helpers/http_helpers.ml

```diff
@@ -23,7 +23,15 @@ module Log = (val Logs.src_log src : Logs.LOG)
 (* Types *)
 (*****************************************************************************)
 
-type get_info = { response : Cohttp.Response.t; code : int }
+type server_result = (string, string) result
+
+type server_response = {
+  body : server_result;
+  response : Cohttp.Response.t;
+  code : int;
+}
+
+type client_result = (server_response, string) result
```

Comment:
`client_result` containing a `server_result` feels odd to me. I think `server_result` needs more documentation relating to when it is `Error _` to help clarify the relationship–is that just when we get a 400/500 status code? It sounds like it may only be `Ok _` when we get a 200.

Reply:
I'll rename server result, and add docs